File: tests/fixtures/java-spring/src/main/java/com/example/controller/OrderController.java
package com.example.controller;

import com.example.service.OrderService;

@org.springframework.web.bind.annotation.RestController
public class OrderController {
    private final OrderService orderService;

    public OrderController(OrderService orderService) {
        this.orderService = orderService;
    }

    public Object queryByStoreNo(String storeNo) {
        return orderService.findByStoreNo(storeNo);
    }
}

File: tests/fixtures/csharp-ef/src/Controllers/OrderController.cs
using Example.Services;
using Example.Data;

namespace Example.Controllers
{
    public class OrderController
    {
        private OrderService service;

        public OrderController(OrderService service)
        {
            this.service = service;
        }

        public Order Get(string storeNo)
        {
            return service.FindByStoreNo(storeNo);
        }
    }
}
